go/go.mod

```diff
@@ -24,16 +24,13 @@ require (
 	github.com/eapache/channels v1.1.0
 	github.com/fxamacker/cbor/v2 v2.2.1-0.20200526031912-58b82b5bfc05
 	github.com/go-kit/kit v0.10.0
-	github.com/golang/protobuf v1.4.0
+	github.com/golang/protobuf v1.4.2
 	github.com/golang/snappy v0.0.1
 	github.com/grpc-ecosystem/go-grpc-middleware v1.0.1-0.20190118093823-f849b5445de4
 	github.com/hpcloud/tail v1.0.0
-	github.com/ipfs/go-log/v2 v2.1.1 // indirect
-	github.com/libp2p/go-libp2p v0.9.6
+	github.com/libp2p/go-libp2p v0.10.0
```

Comment:
It seems like this contains unrelated dependency updates, could you only include changes relevant to this PR?

Reply:
Ok, should be cleaner now.